### PR TITLE
Add linting to travis builds

### PR DESCRIPTION
**Pull request by campionfellin**
This adds a lint phase to the Travis builds.

We also needed to put `tslint` and `typescript` in the `package.json`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,8 @@
     "readline": "^1.3.0",
     "recursive-readdir": "^2.2.1",
     "split-lines": "^1.1.0",
+    "tslint": "^5.9.1",
+    "typescript": "^2.8.1",
```
Comment (grant) [inline package.json]: Can we add these as a dev dependencies?

**Comment by grant** [comment]
Looks good other than the dev dependencies bit.

**Comment by campionfellin** [comment]
@grant looks like Travis build failed due to an erroneous comma in the package.json. Should be fixed now. Should I submit another PR?

**Comment by grant** [comment]
Fixed it.

**Pull request merged**